src/Common/RJSF/common/FieldRow.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { FieldRowProps } from './types'
+
+export const FieldRowWithLabel = ({
+    showLabel,
+    label,
+    required,
+    children,
+    id,
+    shouldAlignCenter = true,
+}: FieldRowProps) => (
+    <div
+        className={
+            showLabel
+                ? `display-grid dc__gap-12 rjsf-form-template__field ${shouldAlignCenter ? 'flex-align-center' : ''}`
+                : ''
+        }
+    >
+        {showLabel && (
```

Comment:
We can add the label check here as well

Reply:
Added a comment for better readability

Reply:
It is required for the grid layout. Added a comment for better readability